src/main/java/com/ciandt/techgallery/service/impl/CronServiceImpl.java

```diff
@@ -166,18 +179,23 @@ public void sendEmailToEndorseds() {
     try {
       List<TechGalleryUser> usersList = techGalleryUserDao.findAll();
       for (TechGalleryUser techGalleryUser : usersList) {
-        List<Endorsement> endorsementsList = endorsementDao.findAllEndorsementsStartingFrom(techGalleryUser,
-            lastExecutedCronJob);
+        List<Endorsement> endorsementsList =
+            endorsementDao.findAllEndorsementsStartingFrom(techGalleryUser, lastExecutedCronJob);
         if (endorsementsList != null) {
-          TechGalleryActivitiesTO activities = new TechGalleryActivitiesTO(Constants.APP_NAME, null, new ArrayList<TechnologyActivitiesTO>());
+          TechGalleryActivitiesTO activities =
```

Comment:
@brunoliberal I think would be great if email TOs name be related to email. I mean something like TechGalleryActivitiesEmailTemplateTO. Also put it in an email TO package.